tests/schema/tests.py

```diff
@@ -1573,11 +1573,10 @@ class Meta:
         new_field = CharField(max_length=255, unique=True)
         new_field.model = Author
         new_field.set_attributes_from_name('name')
-        with patch_logger('django.db.backends.schema', 'debug') as logger_calls:
+        with self.assertLogs('django.db.backends.schema', 'DEBUG'):
+            # One SQL statement is executed to alter the field.
```

Comment:
This makes the test too lax. The documentation for `assertLogs()` says, "The test passes if at least one message emitted inside the with block matches the logger and level conditions, otherwise it fails." A regression that results in multiple logger calls/SQL statements will still pass the test. (Of course, this issue applies everywhere that's changed, but it's particularly important here.)